gitman/commands.py

```diff
@@ -183,6 +183,7 @@ def delete(*, root=None, force=False):
         common.dedent(level=0)
         common.show("Deleting all dependencies...", color='message', log=False)
         common.show()
+        shell.cd(root)
```

Comment:
Why is this needed? I'd rather the API functions not need to interact with the shell directly.